crates/core/src/test_utils/deploy_helpers.rs

```diff
@@ -390,6 +372,58 @@ async fn compute_starknet_address(
     *eoa_account_starknet_address_result.unwrap().first().unwrap()
 }
 
+pub fn compute_kakarot_contracts_class_hash() -> Vec<(String, FieldElement)> {
+    dotenv().ok();
```

Comment:
should the `dotenv()` not be in `compiled_kakarot_paths`?

Reply:
Yeah that's better, moved it 🤙